src/views/collection-view/helpers/get-slug.ts

```diff
@@ -19,7 +19,10 @@ export function getTutorialSlug(
 	const tutorialFilename = splitProductFromFilename(tutorialDbSlug)
 
 	// @TODO genericize this to use 'topic' or 'section' instead of 'product'
-	if (rawProductSlug === 'well-architected-framework') {
+	if (
+		rawProductSlug === 'well-architected-framework' ||
+		rawProductSlug === 'onboarding'
```

Comment:
Accounting for onboarding here too now since that was failing the tutorial-map build 🤷

Reply:
TODO, remove this